app/models/company.rb

```diff
@@ -16,6 +16,14 @@ class Company < ApplicationRecord
 
   belongs_to :region
 
+  # must qualify name with 'company' because there are other tables that use 'name' and if
+  # this scope is combined with a clause for a different table that also uses 'name',
+  # SQL won't know which table to get 'name' from
+  #  name could be NULL or it could be an empty string
+  #  old_region could be NULL or it could be and empty string
+  scope :complete, -> { where('"companies"."name" <> :blank_name AND (region_id IS NOT NULL OR (old_region IS NOT NULL AND old_region <> :blank_old_region))',
+                              {blank_name: '', blank_old_region: ''}) }
```

Comment:
I think that we may not need to reference "old_region" anymore.  I was using that reference in code after we switched to the new region (company ```belongs_to :region```) just in case any prod records did not have accurate ```region``` attributes (by which we built the ```belongs_to``` association).

I think that all old records were changed over to the new structure.  (I would verify this but I don't currently have access to the DO instance).  

If so, we should stop perpetuating the old_region references.   (Also, I need to add a story to remove that field from the DB and all code references to that).